blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/BitbucketOrg.java

```diff
@@ -48,12 +47,12 @@ public String getAvatar() {
     @Override
     public boolean isJenkinsOrganizationPipeline() {
         for(TopLevelItem item: Jenkins.getInstance().getItems()){
-            if(item instanceof OrganizationFolder){
```

Comment:
do we want to completely remove the handling of org folders? what if someone sets them up in classic?

Reply:
I think by handling you mean editing. We only support editing bitbucket repo pipelines created via blueocean. In general classic created org folder pipeline editing won't work. `if(item instanceof OrganizationFolder){` got in bitbucket because of copy paste from GitHub. In case of GitHub due to backward compatibility we support editing of previously create (using blueocean) GitHub org folder.

In fact this method should simply return false.